src/commands/guard/valhalla.ts

```diff
@@ -21,28 +23,90 @@ export const toValhallaExecute = async (
     return;
   }
   try {
-    if (
-      (interaction.channel as TextChannel).parentId ===
-      DISCORD_VALHALLA_CATEGORY_ID
-    ) {
+    const channel = interaction.channel as TextChannel;
+
+    if (channel.parentId === DISCORD_VALHALLA_CATEGORY_ID) {
       const embed = new EmbedBuilder()
         .setColor('#ff3864')
         .setDescription('This is already in Valhalla!');
 
-      await interaction.followUp({ embeds: [embed] });
-    } else {
-      (interaction.channel as TextChannel).setParent(
-        DISCORD_VALHALLA_CATEGORY_ID
+      await sendMessageWithFallback(interaction, channel, embed);
+      return;
+    }
+
+    // Initial response to user
+    const embed = new EmbedBuilder()
+      .setColor('#ff3864')
+      .setDescription(
+        'Starting export process for this channel. This may take a few minutes...'
       );
 
-      const embed = new EmbedBuilder()
+    await sendMessageWithFallback(interaction, channel, embed);
+
+    // Call the Discord Exporter service to start the export
+    try {
+      const exporterUrl =
+        process.env.DISCORD_EXPORTER_URL ||
```

Comment:
Can we move this to `@/utils/constants.ts`? Can we also remove `https://discord-exporter-latest.onrender.com/export` as a backup URL, and instead include a check in the try block?

```
if (!DISCORD_EXPORTER_URL) {
      throw new Error(
        'Missing env DISCORD_EXPORTER_URL'
      );
    }
    ```